Packages/net.yutopp.vgltf.unity/Runtime/MeshImporter.cs

```diff
@@ -36,10 +36,11 @@ public async Task<IndexedResource<Mesh>> Import(int meshIndex, GameObject go, Ca
             var gltf = Context.Container.Gltf;
             var gltfMesh = gltf.Meshes[meshIndex];
 
-            return await Context.Resources.Meshes.GetOrCallAsync(meshIndex, async () =>
-            {
-                return await ForceImport(meshIndex, go, ct);
-            });
+            var mesh = Context.Resources.Meshes.GetOrCall(meshIndex, () => ForceImport(meshIndex));
+
+            var renderer = AddRenderer(go, mesh.Value, gltfMesh);
+            renderer.sharedMaterials = await ImportMaterials(gltfMesh, ct);
```

Comment:
NOTE: Behaviour changes.

current: Material indices are cut if materials are missing. 

PR: Material indices and primitive indices will be same even if materials are missing (null referenced).

Reply:
materials and primitives used to be matching even if missing previously too?
https://github.com/faidra/VGltf/blob/6b211c65838f476582e274190b6b78e4fb26236b/Packages/net.yutopp.vgltf.unity/Runtime/MeshImporter.cs#L134
https://github.com/faidra/VGltf/blob/6b211c65838f476582e274190b6b78e4fb26236b/Packages/net.yutopp.vgltf.unity/Runtime/MeshImporter.cs#L159

Reply:
Thank you for your pointing it out... (You're correct 🙏 )